remote-api/src/main/java/org/alfresco/rest/api/impl/CategoriesImpl.java

```diff
@@ -128,6 +134,10 @@ public List<Category> createSubcategories(final StoreRef storeRef, final String
                     {
                         category.setCount(0);
                     }
+                    if (parameters.getInclude().contains(INCLUDE_PATH_PARAM))
+                    {
+                        category.setPath(nodeService.getPath(nodes.getNode(category.getId()).getNodeRef()).toDisplayPath(nodeService, permissionService));
```

Comment:
I believe we could extract the get path part to a separate method to be less DRY and more readable here (and everywhere you used `setPath`).

Reply:
Extracted